primer.md

```diff
@@ -112,11 +112,36 @@ specifications of how to serialize the event in different formats and
 transports (e.g. JSON and HTTP).
 
 ### Non-Goals
-The following will not be part of the specification:
+The following are considered beyond the scope of the specification:
 * Function build and invocation process
 * Language-specific runtime APIs
 * Selecting a single identity/access control system
 
+The following are features that the working group has explicitly decided
+should nto be part of the specification:
+* **routing information** The CloudEvents spec will never include routing
+   information (e.g. a Pub/Sub topic over which the message is delivered).
+   This is a common suggestion by those new to the concepts of CloudEvents.
+   After much deliberation, the working group has come to the conclusion that
+   routing is unnecessary in the spec: any transport protocol (e.g. HTTPS,
+   MQTT, XMPP, or a Pub/Sub bus) already defines semantics for routing.
+   For example, the Cloud Events [HTTPS binding](http-transport-binding.md)
+   dictates headers and request body contents. CloudEvents don't need to
+   include a destination URL in the spec to be HTTP compatible; the HTTP spec
+   already includes one in the
+   [Request-Line](https://tools.ietf.org/html/rfc2616#section-5.1).
+   
+   Routing information is not just redundant, it detracts.
+   CloudEvents should increase interoperability and decouple the producer and
+   consumer of events. Prohibiting routing information from the events format
+   allows CloudEvents to be redelivered to new actions to be delivered
```

Comment:
I think you're missing something in this sentence. Perhaps ", or " after "actions" perhaps?